### PR TITLE
Fabo/twitter images

### PR DESCRIPTION
**Pull request by faboweb**
Poll Twitter images for Polkadot validators

### CODE REVIEW

File: scriptRunner/src/twitterImages.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+const fetch = require("node-fetch")
+const database = require("../../api/lib/database")
+const config = require("../../api/config")
+const db = database(config)
+
+module.exports.getTwitterImages = async () => {
+    const networks = await db("").getNetworks()
+    const cosmosNetworks = networks.filter(({network_type}) => network_type === "polkadot")
```
Comment (jbibla) [inline scriptRunner/src/twitterImages.js:8]: shouldn't this be `substrateNetworks` or `polkadotNetworks`?

Comment (faboweb) [reply scriptRunner/src/twitterImages.js:8]: There is a suggestion function :P

Comment (jbibla) [reply scriptRunner/src/twitterImages.js]: i wasn't sure which one you'd prefer 😉

Comment (faboweb) [reply scriptRunner/src/twitterImages.js]: sure... ;P

**Comment by faboweb** [comment]
Needs to add the twitter API key to the deployment still

**Pull request merged**